lib/trento/application/integration/discovery/policies/sap_system_policy.ex

```diff
@@ -121,6 +121,8 @@ defmodule Trento.Integration.Discovery.SapSystemPolicy do
          },
          host_id
        ) do
+    cluster_id = Trento.Clusters.get_cluster_id_by_host_id(host_id)
```

Comment:
Would it be better to run this in the `discovery.ex` file?
We do the same to get the current instances, so the policy stays 100% pure, without external side-effects

Reply:
Done